### DISCUSSION TITLE
Event-Calendar

### DISCUSSION THREAD

**Discussion by Euler-B** [Golang-Venezuela/adan-bot#29] [Ideas]
We are planning how it should be and what technical requirements the **Event-Calendar** functionality should have.

**Comment by daniklean** [comment]
I think the user inserts his date `dd/mm/yy` or `mm/dd/yy` since the user ID comes from the context, then it is inserted into a database, that value when equal to it (taking out the UTC and taken to a specific time zone) . ) a trigger is activated. What I'm thinking is how to activate the trigger, from the same database?

> [!TIP]
> Command: /hb nn/nn/nn

/hb nn/nn/nn - Process > save

The thing is when it gets triggered from the db
